README.md

```diff
@@ -501,6 +501,55 @@ The IP configuration supports the following options:
     Note that Classless inter-domain routing (CIDR) notation or network mask notation
     are not supported yet.
 
+- `routing_rule`
+
+    The policy routing rules can be specified via a list of rules given in the
+    `routing_rule` option, which allow routing the packets on other packet fields
+    except for destination address. The default value is a an empty list. Each rule is
+    a dictionary with the following entries:
+   - `priority` -
+      The priority of the rule. A valid priority ranges from 0 to 4294967295. Higher
+      number means lower priority.
+   - `action` -
+      The action of the rule. The possible values are `to-table` (default),
+      `blackhole`, `prohibit`, `unreachable`.
+   - `dport`-
+      The range of the destination port. The range consists of start and end, which can
+      be specified in the form of `$start - $end`. The valid range value has to falls
+      within the range `0 - 65534`.
```

Comment:
The priority uses a different format for the range using "to" which looks better IMHO. Also using $start and $end does not seem to be the style that is used for the documentation here. Does the destination port always need to be a range?

Additionally, I mentioned already on the BZ regarding the acceptance criteria that it would be better IMHO not to use these abbreviations like `dport` to make it easier to read.

Reply:
> The priority uses a different format for the range using "to" which looks better IMHO. 
Do you mean to use `to` instead of `-` in between the range start and range end ?

Reply:
I changed into the following, is this better ?
```
   - `dport`-
      The range of the destination port. The range consists of start, hyphen and end.
      The valid range value has to falls within the range `0 - 65534`.
```